apis/cpi/v1alpha1/vspherecpiconfig_types.go

```diff
@@ -70,20 +70,16 @@ type NSXT struct {
 	// +kubebuilder:validation:Optional
 	RootCAData string `json:"rootCAData"`
 
-	// The name of secret that stores CPI configuration
-	// +kubebuilder:validation:Optional
-	SecretName string `json:"secretName"`
-
-	// The namespace of secret that stores CPI configuration
+	// The secret that stores CPI configuration
 	// +kubebuilder:validation:Optional
-	SecretNamespace string `json:"secretNamespace"`
+	CPISecretRef *v1.SecretReference `json:"cpiSecretRef"`
```

Comment:
What's this?

Reply:
removed. Currently only keep `CredentialRef` as the only source of NSXT credential.